package.json

```diff
@@ -22,6 +23,7 @@
     "url": "git@github.com:AppGeo/wms.git"
   },
   "devDependencies": {
+    "eslint": "^3.7.1",
```

Comment:
while we are at it we should probably fix the eslint because it doesn't currently work I don't think